### PR TITLE
ISPN-10673 Remove Cross-Site Replication from Local caches

### PR DESCRIPTION
**Pull request by pruivo**
https://issues.redhat.com/browse/ISPN-10673

### CODE REVIEW

File: core/src/test/java/org/infinispan/xsite/XSiteCacheConfigurationTest.java [hunk 1]
```diff
@@ -73,18 +76,20 @@ public void testSameBackupDefinedMultipleTimes() {
       cb.build();
    }
 
-   @Test(expectedExceptions = CacheConfigurationException.class)
+   @Test(expectedExceptions = CacheConfigurationException.class, expectedExceptionsMessageRegExp = "ISPN\\d+: The 'site' must be specified!")
```
Comment (karesti) [inline core/src/test/java/org/infinispan/xsite/XSiteCacheConfigurationTest.java]: Could this be refactored so instead using the @ expected exceptions we use the Exceptions class ? while I'm writing this comment I'm not sure anymore if we want to use the expectedExceptions in the annotation or the org.infinispan.commons.test.Exceptions class. 
Could you please tell us which is the best way or both ways are equally valid now @danberindei ?

Comment (pruivo) [reply core/src/test/java/org/infinispan/xsite/XSiteCacheConfigurationTest.java]: I can change to whatever version you want :)

**Comment by karesti** [comment]
@pruivo need rebase

**Comment by pruivo** [comment]
rebased!

**Comment by gustavonalle** [comment]
Needs rebase again

**Comment by pruivo** [comment]
updated!

**Pull request merged**